homeassistant/components/fibaro.py

```diff
@@ -348,6 +348,11 @@ def name(self):
         """Return the name of the device."""
         return self._name
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return self.ha_id
```

Comment:
The unique ID must never change, like a serial number or a MAC address. I don't think that is true for this value?